Compile error on check

**Bug description**
Compilation error on this code:
```
libs/libgui/src/settings/generalconfigwidget.cpp:
  #ifdef NO_UPDATE_CHECK                                                          
      check_upd_chk->setChecked(false);                                           
      check_upd_chk->setVisible(false);                                           
  #endif 
```
```
src/settings/generalconfigwidget.cpp: In constructor ‘GeneralConfigWidget::GeneralConfigWidget(QWidget*)’:
src/settings/generalconfigwidget.cpp:224:9: error: ‘check_upd_chk’ was not declared in this scope; did you mean ‘check_update_chk’?
  224 |         check_upd_chk->setChecked(false);
      |         ^~~~~~~~~~~~~
      |         check_update_chk
```
I'm compiling pgmodeler with NO_UPDATE_CHECK set, obviously.

**Info about your desktop**
 - OS: Linux
 - Version: Archlinux, up to date
 - pgModeler version: 0.9.4 beta
 - Qt version: 5.15.2

This patch fixes it, do you want me to do a PR ?
```
diff -r -c pgmodeler-0.9.4-beta/libs/libgui/src/settings/generalconfigwidget.cpp pgmodeler-0.9.4-beta-new/libs/libgui/src/settings/generalconfigwidget.cpp
*** pgmodeler-0.9.4-beta/libs/libgui/src/settings/generalconfigwidget.cpp   2021-09-20 16:08:24.000000000 +0200
--- pgmodeler-0.9.4-beta-new/libs/libgui/src/settings/generalconfigwidget.cpp   2021-09-22 07:37:04.647917909 +0200
***************                                                                 
*** 221,228 ****                                                                
    });                                                                         
                                                                                
  #ifdef NO_UPDATE_CHECK                                                        
!   check_upd_chk->setChecked(false);                                           
!   check_upd_chk->setVisible(false);                                           
  #endif                                                                        
                                                                                
    //Retrieving the available UI dictionaries                                  
--- 221,228 ----                                                                
    });                                                                         
                                                                                
  #ifdef NO_UPDATE_CHECK                                                        
!   check_update_chk->setChecked(false);                                        
!   check_update_chk->setVisible(false);                                        
  #endif                                                                        
                                                                                
    //Retrieving the available UI dictionaries 
 ```

--- a/libs/libgui/src/settings/generalconfigwidget.cpp
+++ b/libs/libgui/src/settings/generalconfigwidget.cpp
@@ -221,8 +221,8 @@ GeneralConfigWidget::GeneralConfigWidget(QWidget * parent) : BaseConfigWidget(pa
 	});
 
 #ifdef NO_UPDATE_CHECK
-	check_upd_chk->setChecked(false);
-	check_upd_chk->setVisible(false);
+	check_update_chk->setChecked(false);
+	check_update_chk->setVisible(false);
 #endif
 
 	//Retrieving the available UI dictionaries

